src/components/MessageListItem.react.js

```diff
@@ -56,6 +57,8 @@ class MessageListItem extends React.Component {
     let {message} = this.props;
     let stringWithLinks = this.props.message.text;
     let replacedText = parseAndReplace(stringWithLinks);
+    let messageContainerClasses  = cx('message-container '
```

Comment:
I mean, you are passing a concatenated string to cx, so cx would simple return the same string. 
I'd prefer using cx in more complex scenarios.

By why do you want to add a class from authorName ?